### ISSUE TITLE
A very long response

### ISSUE THREAD

**Issue by Hedgehogues** [tiangolo/fastapi#699]
I have a service. After start, my service has response in a second. Some minutes later, my service has response in 10 ms. Why this case appear?

Notice. This is property of service by fastapi, as I measure time of execution of my handler into my code. This is 10 ms.

**Comment by sm-Fifteen** [comment]
It's hard to tell without any code example, but slowdowns with that level of variation usually come from databases or other external network operations. I've had request go from 2 minutes to 15 seconds just by running it a few more times, probably because the database I'm connected to keeps recently accessed data in cache, making query execution and row fetching time much faster when accessing the same data repeatedly.

That would be my guess, anyway. Is there anymore information you can provide on your service? A simplified code snippet, maybe?

**Comment by dmontagu** [comment]
Yeah, without more code it would be very hard to assess, but as far as I'm aware there is no systemic issue with slow response times shortly after app launch.

**Comment by tiangolo** [comment]
Thanks for the help here @sm-Fifteen and @dmontagu ! :cake: :bow:
